src/components/dashboard/container/SpecificConcept.jsx

```diff
@@ -69,6 +68,16 @@ export class SpecificConcept extends Component {
     }));
   }
 
+  handlePath = (path, dictionarName) => {
```

Comment:
could you correct the grammatical error on line 71 dictionarName to dictionaryName

Reply:
oh thanks